sites/stackoverflow.com/answers.md

```diff
@@ -1,95 +1,74 @@
+###[A] Link only (SO link)
+This should be a comment, not an answer. If it is a duplicate question, [vote to close](//stackoverflow.com/help/privileges/close-questions) as such and/or leave a comment once you [earn](http://meta.stackoverflow.com/q/146472) enough [reputation](//stackoverflow.com/help/whats-reputation). If not, *tailor the answer to this specific question*.
 
-###[A] Link only to other SE
+###[A] Link only (SE link)
 Please don't post link-only answers to other Stack Exchange questions. Instead, include the essential portions of the answer here, and *tailor the answer to this specific question.*
 
-###[A] Link only answer
-While this link might provide some limited, immediate help, an answer [should include sufficient context around the link](//meta.stackoverflow.com/a/8259) so your fellow users will have some idea what it is and why it’s there. Always quote the most relevant part of an important link, to make it more useful to future readers with other, similar questions. In addition, other users tend to respond negatively to answers which are [barely more than a link to an external site](//meta.stackexchange.com/q/225370), and they [might be deleted](//$SITEURL$/help/deleted-answers).
+###[A] Link only
+A link to a solution is welcome, but please ensure your answer is useful without it: [add context around the link](//meta.stackexchange.com/a/8259) so your fellow users will have some idea what it is and why it’s there, then quote the most relevant part of the page you're linking to in case the target page is unavailable. [Answers that are little more than a link may be deleted](//$SITEURL$/help/deleted-answers).
 
-###[A] Answering off-topic / bad question
-Please don't post answers on obviously off topic / bad questions! [See: **Should one advise on off topic questions?**](//meta.stackoverflow.com/q/276572)
+###[A] Answering off-topic/bad question
+Please don't post answers on obviously off topic/bad questions! [See: **Should one advise on off topic questions?**](//meta.stackoverflow.com/q/276572)
 
 ###[A] Tool recommendation for a tool request
 Thank you for participating on Stack Overflow. However please note that the question you have answered is off-topic as per the [help/on-topic] since it is a tool request. Answers to such questions tend to be of [poor quality](//meta.stackexchange.com/a/8259) and quickly become stale (what might have been a good resource two years ago may simply not exist anymore). You can flag this question as off-topic by clicking the "flag" link once you have sufficient reputation.
 
 ###[A] Pointing out a typo
 Please don't post answers only pointing out a typographical issue or a missing character. Such answers are unlikely to help future visitors since they are specific to OP's code. Instead, flag or vote to close the question as off-topic as per the [help/on-topic].
 
-###[A] Code as image
-[Please don't post your code as an image](//meta.stackoverflow.com/q/285551).
+###[A] Data as image
```

Comment:
Thanks for the contributions so far! Really nice that this repo get a facelift.

This is really the only change I have a mild issue with. It's changing the title, which people (myself included) use to search quickly for the comment I need.

The downside of this entire auto-comments syncing system is that you are never notified when your entries get modified. That's ok if the comment text is changed, but the title is how you search for stuff. If I didn't know about this change and I went to search for `code as...` I wouldn't get what I was looking for.

"Oh, it's called "DATA as image" now?!?!?" I can imagine someone saying.

Reply:
@gunr2171 I am gonna change that to `*/Code as image` to avoid confusions.